core/proofs/proofs.hpp

```diff
@@ -280,6 +280,8 @@ namespace fc::proofs {
    private:
     static fc::common::Logger logger_;
   };
+
+  UnpaddedPieceSize padPiece(const std::string &path);
```

Comment:
Maybe move it to up, because can be lost under the huge class

Reply:
We already have `proofs` namespace, `Proofs` static methods are like namespace

Reply:
Just for readability